spec/menu_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'menu'
+
+class MenuUser; include Menu; end
+
+describe 'menu' do
+
+let(:subject) {MenuUser.new}
+let(:menu) {menu.menu}
+
+it 'has a menu' do
+    expect(subject.menu).to be_kind_of Array
+  end
```

Comment:
Lint/BlockAlignment: `end` at 12, 2 is not aligned with `it 'has a menu' do` at 10, 0